### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ivarpruijn**
Updates to be more inline with latest site updates and releases.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,75 +1,62 @@
...
+It's designed and tested with UIs and analytics use cases of over 50 companies. Areas like payments & CRM are on the roadmap, and it's extensible to cover custom use cases.
 
-### Tracking SDK
+### Tracking SDKs
 
 Supports front-end engineers to [implement tracking instrumentation](https://www.objectiv.io/docs/tracking) that embraces the open analytics taxonomy.
 
-* Provides guidance, validation & E2E esting to help setting up error-free instrumentation
-* Support for React, React Native, Angular and Browser
+* Provides validation and end-to-end testing tooling to set up error-free instrumentation.
```
Comment (vincenthoogsteder) [inline README.md:36]: @ivarpruijn link to  https://objectiv.io/docs/tracking/core-concepts/validation?

File: README.md [hunk 2]
```diff
@@ -1,75 +1,62 @@
...
  
 ### Open model hub
 
-A [growing collection of pre-built models and functions](https://objectiv.io/docs/modeling/open-model-hub/) to run, combine or customize for quick, in-depth analyses.
+A [growing collection of pre-built models and functions](https://objectiv.io/docs/modeling/open-model-hub/), so you can take and run what someone else made, or quickly build your own with pre-built models and functions.
 
-* Covers a wide range of use cases: from basic product analytics to predictive analysis with ML
-* Works with any dataset that embraces the open analytics taxonomy
-* New models & functions are added continuously
+* Covers a wide range of use cases: from basic product analytics to predictive analysis with ML.
```
Comment (vincenthoogsteder) [inline README.md:43]: @ivarpruijn link to https://objectiv.io/docs/modeling/example-notebooks/?

File: README.md [hunk 3]
```diff
@@ -1,75 +1,62 @@
...
+* [Objectiv Docs](https://www.objectiv.io/docs) - Technical documentation.
+* [Objectiv on Slack](https://objectiv.io/join-slack) - Get help & join the discussion.
+* [Objectiv.io](https://www.objectiv.io) - Our website.
 
-### Resources
-
-* [Objectiv Docs](https://www.objectiv.io/docs) - Objectiv's official documentation.
-* [Objectiv on Slack](https://objectiv.io/join-slack) - Get help & join the discussion on where to take Objectiv next.
-* [Objectiv.io](https://www.objectiv.io) - Objectiv's official website.
+If you like what we do, give us a star!
```
Comment (vincenthoogsteder) [inline README.md:15]: @ivarpruijn maybe we add a short section below on Getting started, pip install objectiv-modelhub (zo it's clear it's a lib) and link to https://objectiv.io/docs/modeling/get-started-in-your-notebook/?

**Review by vincenthoogsteder — COMMENTED** [review]
see comments

**Review by vincenthoogsteder — APPROVED** [review]
LGTM

File: README.md (hunk 3 above)
Comment (ivarpruijn) [reply README.md:15]: done

**Pull request merged**